### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by Diegogagan2587**
# Implemented Changes
- Create a table named owners with the following columns:
  - id: integer (set it as autoincremented PRIMARY KEY)
  - full_name: string
  - age: integer image

- Create a table named species with the following columns:
  - id: integer (set it as autoincremented PRIMARY KEY)
  - name: string 
image

- Modify animals table:
  - Make sure that id is set as autoincremented PRIMARY KEY
  - Remove column species
  - Add column species_id which is a foreign key referencing species table
  - Add column owner_id which is a foreign key referencing the owners table  image

- Insert data into owners 
 image

- Insert Data into species:
image

- Insert the following data into the species table:
  - Pokemon
  - Digimon 
image
- Modify your inserted animals so it includes the species_id value:
  - If the name ends in "mon" it will be Digimon
  - All other animals are Pokemon 
image

- Modify your inserted animals to include owner information (owner_id):
  - Sam Smith owns Agumon.
  - Jennifer Orwell owns Gabumon and Pikachu.
  - Bob owns Devimon and Plantmon.
  - Melody Pond owns Charmander, Squirtle, and Blossom.
  - Dean Winchester owns Angemon and Boarmon. 
image

- Write queries (using JOIN) to answer the following questions:
  - What animals belong to Melody Pond? 
image

  - List of all animals that are Pokemon (their type is Pokemon). 
image

  - List all owners and their animals, remember to include those that don't own any animal. 
image

  - How many animals are there per species? 
image

  - List all Digimon owned by Jennifer Orwell. 
image

  - List all animals owned by Dean Winchester that haven't tried to escape. 
image

  - Who owns the most animals? 
image

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -31,4 +31,28 @@ INSERT INTO animals (name, date_of_birth, weight_kg, neutered,scape_attempts)
...
+UPDATE animals SET species_id = 2 WHERE name LIKE '%mon%';
+UPDATE animals SET species_id = 1 WHERE species_id IS NULL;
+SELECT * FROM animals;
+
+UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
+UPDATE animals SET owner_id = 2 WHERE name IN ('Gabumon', 'Pikachu');
+UPDATE animals SET owner_id = 3 WHERE name IN ('Devimon', 'Plantmon');
+UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander','Squirtle','Blossom');
+UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon','Boarmon');
+
```
Comment (Qoosim) [inline data.sql:57]: - [ ] It is required to write queries (using JOIN) to answer the some questions according to the project requirements. Kindly check the project requirements and implement your queries. 

### Project Requirements.
prj-reqt

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @Diegogagan2587, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-  Correct Gitflow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly remove the live demo link from your readme file as this milestone does not require deployment.
- [ ] Some headers in your readme file do not have appropriate information under them. Kindly provide them to make your readme file look professional.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (Diegogagan2587) [reply data.sql:57]: @Qoosim Hi, Hope you are doing Great! I have wrote the queryes within the `queries.sql` file,
image

 those are avaialbe starting on the line 73, Thanks :D!
image

image

Comment (Diegogagan2587) [reply data.sql:57]: @Qoosim Thanks for pointing out the changes I need to do within the Readme, I'll do it now!

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Diegogagan2587 😃, 

Your project is complete!  There is nothing else to say other than... It's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Well done making the required changes from the previous reviewer
- Excellent communication with the reviewer

Cheers, and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**